tests/integration-tests/conftest.py

```diff
@@ -74,6 +74,7 @@ def pytest_addoption(parser):
     parser.addoption(
         "--createami-custom-chef-cookbook", help="url to a custom cookbook package for the createami command"
     )
+    parser.addoption("--createami-cookbook-git-ref", help="Git ref of the cookbook used to bake the AMI")
```

Comment:
can be done in a separate PR but let's name this cookbook-git-ref only and add a similar argument for the node too